src/main/java/spoon/processing/Severity.java

```diff
@@ -17,25 +17,43 @@
 
 package spoon.processing;
 
+import org.apache.log4j.Level;
 import spoon.compiler.Environment;
 
 /**
  * Enumeration that specifies the severity of the message reported by processors
- * 
+ *
  * @see Environment#report(Processor, Severity,
- *      spoon.reflect.declaration.CtElement, String)
+ * spoon.reflect.declaration.CtElement, String)
  */
 public enum Severity {
 	/**
 	 * Error-level severity.
 	 */
-	ERROR,
+	ERROR {
```

Comment:
btw, I don't like this enum. why overriding the toLevel method for every enum value? You could write ```ERROR(Level.ERROR)``` and add a constructor taking the Level as an argument. This way you don't have to override the method 3 times. you can write it once and use it for as many as enum values as you want. 

```java
private final Level level;

Severity(Level level) {
    this.level = level;
}

public Level toLevel() {
   return this.level;
}
```